src/components/fabSpeedDial/fabController.js

```diff
@@ -135,9 +135,13 @@
 
     // Fire the animations once in a separate digest loop to initialize them
     function fireInitialAnimations() {
-      $mdUtil.nextTick(function() {
-        $animate.addClass($element, 'md-noop');
-      });
+      var delay = 100; // Delay until after the element has been appened to the DOM
```

Comment:
I'm really not a fan of hard-coded delays like this. Is there any other way to approach this? I can consult with @matsko if it's an issue around ngAnimate.